code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -235,13 +235,19 @@
 	item_state = "mime_envirohelm"
 	visor_icon = "mime_envisor"
 
+/obj/item/clothing/head/helmet/space/plasmaman/mime/nodrop
+	flags = NODROP
```

Comment:
make sure this doesn't create the same problem as the original `nodrop` varients of the mime and clown mask had.

Reply:
It doesn't.

Reply:
It will since the flags get overriden here